ui.R

```diff
@@ -855,7 +855,7 @@ dashboardPage(
             # collapsible = TRUE,
             # collapsed = TRUE,
             width = 12,
-            htmlTableWidgetOutput("changelog")
+            dataTableOutput("changelog")
```

Comment:
Yay, this is great, the new table looks great! I think the other thing we need to do here is remove "library(htmlTable)" from global.R and remove it from the renv snapshot as well. Thanks so much for making this work @zabalad! Let me or Alex know if you need help with renv